articles/azure/azs-how-use-azure-api.md

```diff
@@ -112,36 +136,54 @@ To obtain an access token:
       "ext_expires_in": "0",
       "expires_on": "1512574780",
       "not_before": "1512570880",
-      "resource": "https://management.ukcloud.onmicrosoft.com/c0a3a144-57ae-42e2-a5da-19e945a25deb",
-      "access_token": "eyJ0eXAiOi...truncated for readability..."
+      "resource": "https://management.as2ukcloud.onmicrosoft.com/90ada28c-5aed-4248-90c7-0538504217f1",
+      "access_token": "eyJ0eXAiOi...truncated for readability...",
+      "refresh_token": "AQABAAAAAA...truncated for readability..."
     }
     ```
 
-5. You must include this token in the Authorization header of each subsequent API request. For example:
+5. You must include this token in the Authorization header of each subsequent API request, for example:
+
+    ### [Bash](#tab/tabid-3)
```

Comment:
I think you can use the same tabid for this as for the Bash tab above, so if someone selects the Bash tab in the previous section, it will be selected by default in this section (saves them having to select the tab in each section)

Reply:
😱 Awesome, thanks Sue 👍 😄